web/src/components/merchant/listing-details/CommittedCustomerStatuses.tsx

```diff
@@ -50,28 +54,43 @@ const CommitStatusCount = styled.div`
   font-weight: bold;
 `;
 
+const CommitStatusDetails = styled.div`
+  margin-left: 15px;
+  margin-top: 10px;
+  max-height: 60vh;
+`;
+
 interface CommitStatusStatsProps {
   colour: 'bright-red' | 'yellow' | 'green';
```

Comment:
Personally prefer to standardise with `color`. Not too particular with language but I feel that because we are using `color` for css we should just standardise so we don't have both in the repo!
```suggestion
  color: 'bright-red' | 'yellow' | 'green';
```

Reply:
@hzxcaryn Okay! Was also considering this when I made this property name, because in general we use British spelling which means should use 'colour' but then css uses American spelling i.e. 'color'. :/